plugin/plugins/peerlog/peerlog.go

```diff
@@ -41,60 +67,123 @@ func (*peerLogPlugin) Version() string {
 }
 
 // Init initializes plugin
-func (*peerLogPlugin) Init(*plugin.Environment) error {
+func (pl *peerLogPlugin) Init(*plugin.Environment) error {
+	pl.events = make(chan plEvent, eventQueueSize)
 	return nil
 }
 
-func (*peerLogPlugin) Start(node *core.IpfsNode) error {
+func (pl *peerLogPlugin) collectEvents(node *core.IpfsNode) {
+	go func() {
```

Comment:
why not call `go collectEvents()` and skip the inner anonymous function?

Reply:
Oops. Thanks!